HomePage.cs

```diff
@@ -18,28 +18,25 @@ public void startBrowser()
         {
             driver = new ChromeDriver();
             driver.Manage().Window.Maximize();
+            driver.Url = "https://www.saucedemo.com/";
```

Comment:
It's better to have such values as constants in one place, ideally config file so it can be easily managed when needed.